src/utils/featheryClient/index.ts

```diff
@@ -86,14 +89,24 @@ export default class FeatheryClient extends IntegrationClient {
     };
     if (collaboratorId) data.collaborator_user = collaboratorId;
 
-    const options = {
+    const options: RequestOptions = {
       headers: {
         'Content-Type': 'application/json'
       },
       method: 'POST',
       body: JSON.stringify(data)
     };
-    return this._fetch(url, options);
+    if (navigator.onLine) {
+      return this._fetch(url, options);
+    } else {
+      const { sdkKey } = initInfo();
+      options.headers = {
```

Comment:
options.headers.Authorization = ...

if your'e running into a type error, you could do something like 

headers: {'Content-Type': 'application/json'} as Record<string, string> in the initial option definition